dask/array/core.py

```diff
@@ -1047,6 +1047,19 @@ def _set_chunks(self, chunks):
     def __len__(self):
         return sum(self.chunks[0])
 
+    def __array_ufunc__(self, numpy_ufunc, method, *inputs, **kwargs):
+        import pdb; pdb.set_trace()
+        from . import ufunc
+        da_ufunc = getattr(ufunc, numpy_ufunc.__name__)
+        if method == '__call__':
+            return da_ufunc(*inputs, **kwargs)
```

Comment:
I've done this in most cases except those where the ufunc returns multiple outputs.  In that case I'm reverting back to calling the dask.array version.